go/src/koding/kites/kloud/provider/azure/stack.go

```diff
@@ -316,6 +333,17 @@ func (s *Stack) injectCloudInit(vm map[string]interface{}, name, kiteKeyName str
 	return countKeys, nil
 }
 
+func (s *Stack) id() string {
```

Comment:
Could you make this function part of BaseStack? I would be useful in other providers.